src/signals/incident/components/IncidentClassification/index.js

```diff
@@ -13,20 +15,23 @@ const IncidentClassification = () => {
   const dispatch = useDispatch();
 
   useEffect(() => {
-    get(`${configuration.CATEGORIES_ENDPOINT}${category}/sub_categories/${subcategory}`);
-  }, [category, subcategory, get]);
+    if (isAuthenticated()) {
+      history.push('/');
```

Comment:
Why override setting the category and subcategory for authenticated users? The AC describe that the ML service should still be queried, but not that the category should not be set.

Reply:
This is how the PO has requested, will update the AC